code/modules/atmospherics/machinery/components/unary_devices/heat_exchanger.dm

```diff
@@ -55,8 +55,10 @@
 	var/datum/gas_mixture/air_contents = airs[1]
 	var/datum/gas_mixture/partner_air_contents = partner.airs[1]
 
-	var/air_heat_capacity = air_contents.heat_capacity()
```

Comment:
why remove this

Reply:
true actually, I could've done that much better, moment